__tests__/app.test.js

```diff
@@ -192,35 +192,119 @@ describe("App", () => {
+      return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_value: 1 })
+        .expect(200)
+        .then(({ body, body: { updated_review } }) => {
+          expect(body).toHaveProperty("updated_review");
+          expect(updated_review).toHaveProperty("votes", 2);
+          expect(updated_review).toHaveProperty("owner", expect.any(String));
+          expect(updated_review).toHaveProperty("title", expect.any(String));
+          expect(updated_review).toHaveProperty("review_id", 1);
+          expect(updated_review).toHaveProperty("category", expect.any(String));
+          expect(updated_review).toHaveProperty(
+            "review_img_url",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty(
+            "review_body",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty(
+            "created_at",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty("designer", expect.any(String));
+        });
+    });
+    test("200: Responds with an updated review object with the votes decremented correctly", () => {
+      return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_value: -1 })
+        .expect(200)
+        .then(({ body, body: { updated_review } }) => {
+          expect(body).toHaveProperty("updated_review");
+          expect(updated_review).toHaveProperty("votes", 0);
+          expect(updated_review).toHaveProperty("owner", expect.any(String));
+          expect(updated_review).toHaveProperty("title", expect.any(String));
+          expect(updated_review).toHaveProperty("review_id", 1);
+          expect(updated_review).toHaveProperty("category", expect.any(String));
+          expect(updated_review).toHaveProperty(
+            "review_img_url",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty(
+            "review_body",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty(
+            "created_at",
+            expect.any(String)
+          );
+          expect(updated_review).toHaveProperty("designer", expect.any(String));
+        });
+    });
+    test("404: correct data type for the review_id param, but review is not found", () => {
+      return request(app)
+        .patch("/api/reviews/100")
+        .send({ inc_value: 100 })
+        .expect(404)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Review ID: 100 Not Found");
+        });
+    });
+    test("400: incorrect data type for the review_id param", () => {
+      return request(app)
+        .patch("/api/reviews/abc")
+        .send({ inc_value: 100 })
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Bad Request");
+        });
+    });
+    test("400: incorrect data type for the inc_value property in the request object", () => {
```

Comment:
good 👍 I think that's all the test cases covered, maybe what do you want to happen if they dont send an inc_value on the request, you could just ignore it and say the returned request should be unchanged, but I think it would be a failing test so cold be worth adding